### PR TITLE
[ADD] runbot_travis2docker: Add log-db option

### PR DESCRIPTION
**Pull request by lasley**
- Add env var to log to parent database

Depends:
- [ ] OCA/maintainer-quality-tools#358

Disclaimer - this is theoretical and has not been functionally tested yet

### CODE REVIEW

**Comment by moylop260** [comment]
:+1:

File: runbot_travis2docker/models/runbot_build.py [hunk 1]
```diff
@@ -108,6 +109,11 @@ def job_20_test_all(self, cr, uid, build, lock_path, log_path):
             '--name=' + build.docker_container,
             '-t', build.docker_image,
         ] + pr_cmd_env
+        logdb = cr.dbname
+        if config['db_host'] and grep(build.server('sql_db.py'), 'allow_uri'):
```
Comment (moylop260) [inline runbot_travis2docker/models/runbot_build.py]: We don't have the source code of odoo to `grep` method.
`if config['db_host']:` is enough

**Comment by joaoalf** [comment]
👍 lgtm

**Comment by alan196** [comment]
:+1:

**Comment by lasley** [comment]
Note we have a new commit due to the fact that the Odoo code is not available during the T2D runbot custom build.

This option explicitly deprecates Odoo version 7

**Comment by lasley** [comment]
Latest commit adds a ghetto hack to support 7.0 by branch name. Best I could do, sorry

**Comment by pedrobaeza** [comment]
@lasley, there's a release version that can be queried in Odoo itself: https://github.com/odoo/odoo/blob/10dcec1e9af583e2a41acbe3a4676884daf79d76/openerp/release.py#L15

**Comment by lasley** [comment]
@pedrobaeza - We don't have access to the code during the custom build & I cannot identify a way to utilize the branch name from the Travis file.

**Comment by pedrobaeza** [comment]
OK, I see. Then continue this way.

**Review by vincentcoll — APPROVED** [review]

File: runbot_travis2docker/models/runbot_build.py [hunk 2]
```diff
@@ -94,11 +95,13 @@ def job_20_test_all(self, cr, uid, build, lock_path, log_path):
         ] if 'refs/pull/' in build.branch_id.name else [
             '-e', 'TRAVIS_PULL_REQUEST=false',
         ]
+        travis_branch =  build._get_closest_branch_name(
```
Comment (moylop260) [inline runbot_travis2docker/models/runbot_build.py]: Coud you remove the double space?

**Review by moylop260 — APPROVED** [review]

**Comment by lasley** [comment]
@moylop260 done

**Pull request merged**